README.md

```diff
@@ -13,10 +13,43 @@ This repository serves to resolve governance issues and establish shared practic
 
 ### Teams
 
+Each repository should have at least one corresponding [GitHub team](https://github.com/orgs/Meteor-Community-Packages/teams), containing all maintainers of the repository. It is suggested that team's name match repository's name, unless team is used for multiple repositories.
+
+Each repository should also have a corresponding Meteor organization, you can [create it here](https://www.meteor.com/account-settings) under `Organizations` tab. After that you can add Meteor users to it, using web interface, or:
```

Comment:
Should we potentially prefix the org so as to avoid a potential collision? For instance mcg-react-router-ssr

Reply:
Collision with what? I think that organization can be anything because it will be used just for permissions.

We should document names somewhere though. Maybe we can have some file in the repository with such metadata.

Reply:
By collision I just mean to say that there is an off chance that depending on the package name, the org name could be taken. Even possibly be useful to someone else and  not necessarily need to be used for publishing permissions if we could just standardize a prefix. Not saying this is a necessity, just that it's something to consider.

Reply:
For this names I do not suggest to name it the same as the package/repository. Just for GitHub teams.